src/components/ScoreSetHistogram.vue

```diff
@@ -1,4 +1,5 @@
 <template>
+  <!-- This ref name is coupled to parent elements; do not change it. -->
```

Comment:
Might be nicer to expose it as an explicit property. Some developers even discourage fetching subcomponent properties in Vue, but I think it's a reasonable thing to do. Digging into a subcomponent's $refs seems more fragile.

If I remember correctly, one way to do this is to store the ref in a property in mounted(). There may be more elegant ways; and anyway you may decide to leave it as-is.